var/spack/repos/builtin/packages/binutils/package.py

```diff
@@ -129,3 +129,32 @@ def flag_handler(self, name, flags):
             if self.spec.satisfies('@:2.34 %gcc@10:'):
                 flags.append('-fcommon')
         return (flags, None, None)
+
+    def _test_check_versions(self):
+        spec_vers = str(self.spec.version)
+
+        checks = {
+            'ar': spec_vers,
+            'c++filt': spec_vers,
+            'coffdump': spec_vers,
+            'dlltool': spec_vers,
+            'elfedit': spec_vers,
+            'gprof': spec_vers,
+            'ld': spec_vers,
+            'nm': spec_vers,
+            'objdump': spec_vers,
+            'ranlib': spec_vers,
+            'readelf': spec_vers,
+            'size': spec_vers,
+            'strings': spec_vers,
+        }
+
+        for exe in checks:
+            expected = checks[exe]
+            reason = 'test: ensuring version of {0} is {1}' \
+                .format(exe, expected)
+            self.run_test(exe, '--version', expected, installed=True,
+                          purpose=reason, skip_missing=True)
+
+    def test(self):
+        self._test_check_versions()
```

Comment:
What's the point of this helper function?

Reply:
Good point.  It's either because I was used to writing multiple sets of test parts and was using this as a convention, or I had other tests that I removed.  Can't recall.

Reply:
Will remove.  Thanks for catching that.